docs-v2/reference/scripts.mdx

```diff
@@ -726,7 +726,7 @@ await nango.getConnection();
 
 # Sync-specific helper methods
 
-Sync scripts persist data updates to the Nango cache, which your app later fetches (cf. [step-by-step guide](/guides/syncs/use-a-sync)).
+Sync scripts persist data updates to the Nango cache, which your app later fetches (cf. [step-by-step guide](/implementation-guides/syncs/implement-a-sync)).
```

Comment:
should be function no? There is a few other places where it is still called scripts, including the /reference/sdks/scripts.mdx file